python/lsst/meas/base/applyApCorr.py

```diff
@@ -194,11 +194,11 @@ def run(self, catalog, apCorrMap):
                 if self.config.doFlagApCorrFailures:
                     source.set(apCorrInfo.fluxFlagKey, oldFluxFlagState)
 
-            if self.log.getThreshold() <= self.log.DEBUG:
+            if self.log.getLevel() <= self.log.TRACE:
```

Comment:
Why the change from debug to trace? In any case this should match the logging level in the conditional (on the copy I just checked out it does match -- I see `self.log.trace` -- but in this pull request it does not -- I see `self.log.debug` -- so that's a bit of a git puzzle).